server/routes/addCard.js

```diff
@@ -10,8 +10,8 @@ const addCardHandler = (req, res) => {
   });
   if (error === undefined) {
     addCardQuery(userId, productId, quantity)
```

Comment:
You need to change your functions and queries name from `card` to `cart`
```suggestion
    addCartQuery(userId, productId, quantity)
```